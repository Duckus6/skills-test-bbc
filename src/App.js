import React from 'react';
import './App.css';
import ArticleViewer from "./ArticleViewer";

function App() {
    return (
        <div className="App">
            <h3>Header</h3>
            <ArticleViewer/>
        </div>

    );
}

export default App;
