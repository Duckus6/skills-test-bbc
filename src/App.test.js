import React from 'react';
import {shallow} from 'enzyme';
import App from './App';

test('renders', () => {
    const wrapper  = shallow(<App />);
    expect(wrapper.debug()).toMatchSnapshot();
});
