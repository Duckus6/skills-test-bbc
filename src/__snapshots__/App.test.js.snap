// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
"<div className=\\"App\\">
  <h3>
    Header
  </h3>
  <ArticleViewer />
</div>"
`;
